containers/Ideas/Idea.js

```diff
@@ -92,25 +92,24 @@ export const Idea = (props) => {
     setMenuVisible(false)
   );
 
-  const handleVote = async(direction) => {
+  const handleRate = async(value) => {
```

Comment:
@khamui I renamed this, because in Aplus its called rating everywhere and also in the serialized data, so I thought this is more consistent.. I hope that is fine for you, sorry for messing with your code :grimacing: